### DISCUSSION TITLE
Error al ejecutar los comandos maven

### DISCUSSION THREAD

**Discussion by BrianAragon** [dds-jv/foro#400] [Consultas de herramientas] [answered]
Buenas noches,

Al intentar ejecutar los, mvn test, mvn clean verify. Obtengo el siguiente error:
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project ejercicio: Fatal error compiling: error: invalid target release: 17 -> [Help 1]

[INFO] Scanning for projects...
[INFO]
[INFO] -------------------< ar.edu.utn.frba.dds:ejercicio >--------------------
[INFO] Building ejercicio 1.0-SNAPSHOT
[INFO]   from pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- jacoco:0.8.8:prepare-agent (default-prepare-agent) @ ejercicio ---
[INFO] argLine set to -javaagent:C:\\Users\\brian\\.m2\\repository\\org\\jacoco\\org.jacoco.agent\\0.8.8\\org.jacoco.agent-0.8.8-runtime.jar=destfile=C:\\Users\\brian\\OneDrive\\Documentos\\tpi1-macowins-2023-BrianAragon\\target\\jacoco.exec
[INFO]
[INFO] --- resources:3.3.0:resources (default-resources) @ ejercicio ---
[INFO] skip non existing resourceDirectory C:\Users\brian\OneDrive\Documentos\tpi1-macowins-2023-BrianAragon\src\main\resources
[INFO]
[INFO] --- compiler:3.11.0:compile (default-compile) @ ejercicio ---
[INFO] Changes detected - recompiling the module! :source
[INFO] Compiling 1 source file with javac [debug target 17] to target\classes
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.531 s
[INFO] Finished at: 2023-05-06T22:41:35-03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project ejercicio: Fatal error compiling: error: invalid target release: 17 -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

**Comment by flbulgarelli** [comment, marked as answer]
¡Hola!

Lo que el error te está indicando es que no tenes instalada la versión 17 de Java o, al menos, no es el JDK que está siendo utilizado cuando corre maven. 

Asegurate de que que tu variable `JAVA_HOME` apunte a un JDK 17.
